src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -225,7 +229,13 @@ def setUp(self):
 
         #Storage setup if requested in test input file
         if self.nvme_parameter == "nvme":
-            server_utils.storage_prepare(self.hostlist_servers)
+            port = self.params.get("port", "/run/server_config/*")
```

Comment:
The default value handling in lines 233-234 can be implemented here:
```suggestion
            port = self.params.get("port", "/run/server_config/*", "10001")
```